packages/site/pages/components/checkbox.js

```diff
@@ -232,6 +232,20 @@ export default _ => (
         ]}
       />
 
+      <SectionHeading>Indeterminate</SectionHeading>
+      <P>
+        Passing the <Text.Code>indeterminate</Text.Code> indicates that the
+        checkbox value is indeterminate.
```

Comment:
Maybe try to avoid using the word in the definition.  Maybe something like:

"Passing the indeterminate flag indicates that the checkbox selection should be obscured as if the control was in a third, indeterminate, state."